### PR TITLE
LPD-43406 Allow user to localize date fields

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
Related issue: [LPD-43406](https://liferay.atlassian.net/browse/LPD-43406)

[LPD-43406]: https://liferay.atlassian.net/browse/LPD-43406?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
Just started reviewing :)

File: modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java [hunk 1]
```diff
@@ -5069,6 +5174,15 @@ private ObjectFieldSetting _createObjectFieldSetting(
 		return objectFieldSetting;
 	}
 
+	private Timestamp _dateStringToTimestamp(String dateString)
```
Comment (marcelabc) [inline modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java:5177]: Please, rename to `getTimestamp(String dateString)`

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Comment by pedro-oliveira446** [comment]
ci:report:87824340

**Comment by pedro-oliveira446** [comment]
ci:report:87896286

**Pull request closed without merging**